pwa/components/dashboard/home/DashboardHomeContent.tsx

```diff
@@ -66,9 +66,24 @@ export const DashboardHomeContent = ({
     return response['hydra:member'];
   };
 
+  console.log(repairer);
+
   return (
     <Box>
-      {repairer && (
+      {!repairer.enabled && (
+        <Box display={'flex'} justifyContent={'center'}>
+          <Typography
```

Comment:
The following part of message is missing : "Cette boutique est en attente de validation."